distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/callables/GetConfigurationProperties.java

```diff
@@ -65,11 +68,18 @@ public GetConfigurationProperties(ConfigurationAdmin configAdmin, String propert
         String query = String.format("(%s=%s)", propertyName, propertyValue);
         Configuration[] configurations = configAdmin.listConfigurations(query);
 
-        if ((configurations == null) || (configurations.length == 0)) {
+        if (configurations == null) {
             return null;
         }
 
-        if (configurations.length > 1) {
```

Comment:
It isn't 1 anymore. Because I added a new persistence manager, the size will always be at least 2. I just use the set to remove duplicates and then if the size is still greater than 1, we error out.